strategies/categorized/src/test/java/org/arquillian/smart/testing/strategies/categorized/project/FastTaggedClass.java

```diff
@@ -0,0 +1,5 @@
+package org.arquillian.smart.testing.strategies.categorized.project;
```

Comment:
it would be nice to create sub-packages for tagged classes and categorized classes, otherwise, it's quite difficult to orientate there.